lib-satysfi/dist/packages/math.satyh

```diff
@@ -811,6 +812,7 @@ end = struct
   let-math \bigcap = bigop `⋂`
   let-math \bigcup = bigop `⋃`
 
+  let-math \lnot = rel `¬`
```

Comment:
Thanks for improving! Since ¬ is a unary operator, `ord` would be suitable here instead of `rel`.

Reply:
fix this. thanks